docs/content.md

```diff
@@ -4,83 +4,78 @@
 
 ###  Global
 
-* Design Principles
-* Accessibility
-* Voice & tone (off link to Manual of Style?)
-* Designing for Safety
+* Accessibility: consider a11y in design early, list of tools
```

Comment:
In future I think would be good have "inclusive design" guidelines so as to go broader than accessibility standards.

Reply:
I can make it so it's more broad, and call it Inclusive design. When speaking to the a11y consultants last week they were more keen on that term too. It would also work well if we rename this section to "Principles".